src/services/webhook.ts

```
@@ -51,39 +74,81 @@ type WebhookEventPayload = WebhookPayload | UrlVerificationPayload;
 
-	// Use constant-time comparison to prevent timing attacks
-	const a = Buffer.from(computed, 'hex');
-	const b = Buffer.from(receivedSignature, 'hex');
+		const expected = `sha256=${expectedSignature}`;
 
-	// If lengths don't match, signatures are definitely different
-	if (a.length !== b.length) {
+		// Use constant-time comparison to prevent timing attacks
+		const sigBuf = Buffer.from(signature);
+		const expBuf = Buffer.from(expected);
+		
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)